shapes/produce_shapes_2017.py

```diff
@@ -122,7 +121,9 @@ def parse_arguments():
         type=str,
         help="Backend. Use classic or tdf.")
     parser.add_argument(
-        "--tag", default="ERA_CHANNEL", type=str, help="Tag of output files.")
```

Comment:
I personally used the "tag" option, but I guess it is just renamed? If so, please do in all produce_shapes scripts (also 2016 and 2018) so that the options are consistent

Reply:
i switched it back. now it should be "tag" everywhere.